src/components/Navigation.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 
+import {NavLink} from 'react-router-dom';
+
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <NavLink to="/" exact>Home</NavLink>
         </div>
         <div>
-          <a href="">About</a>
+          <NavLink to="/about">About</NavLink>
```

Comment:
Great job using NavLink instead of link. NavLink is great for allowing additional styling options such as breadcrumbs.